metadata-ingestion-modules/airflow-plugin/src/datahub_airflow_plugin/client/airflow_generator.py

```diff
@@ -428,8 +428,8 @@ def run_datajob(
         dpi = DataProcessInstance.from_datajob(
             datajob=datajob,
             id=f"{dag.dag_id}_{ti.task_id}_{dag_run.run_id}",
-            clone_inlets=True,
-            clone_outlets=True,
+            clone_inlets=False if config and not config.materialize_iolets else True,
+            clone_outlets=False if config and not config.materialize_iolets else True,
```

Comment:
These could just be `config.materialize_iolets` in both cases. That is a boolean, right?

Reply:
yes, but implemented this way as there might be a case where `config` is not having `materialize_iolets` as `config` is of type `Optional[DatahubLineageConfig]`

Reply:
Please use `config and not config.materialize_iolets` instead

Reply:
this is generating linter error 
```
src/datahub_airflow_plugin/client/airflow_generator.py:431: error: Argument "clone_inlets" to "from_datajob" of "DataProcessInstance" has incompatible type "Union[DatahubLineageConfig, None, bool]"; expected "bool"  [arg-type]
```

used `config is None or config.materialize_iolets,` instead